src/resource/nonRdfData.test.ts

```diff
@@ -345,7 +346,7 @@ describe("Non-RDF data deletion", () => {
       >;
     };
 
-    fetcher.fetch.mockReturnValue(
+    fetcher.fetch.mockReturnValueOnce(
```

Comment:
Was there a specific motivation to change this ? And should the other instances of mock fetcher be aligned with this ?

Reply:
Yes, since there's a single mock instance of the `../fetcher` module, mocking the return value permanently also affects the other tests. There weren't any before, IIRC, but now that there are, that became problematic. I'll log a ticket to go over the entire code base to see where else this could change, to avoid it becoming an issue in future tests.